api/v1beta1/disruption_webhook.go

```diff
@@ -170,3 +189,135 @@ func (r *Disruption) getMetricsTags() []string {
 
 	return tags
 }
+
+// initialSafetynets runs the initial safety nets for any new disruption
+// returns true if any safety net were caught and returns any errors when attempting to run the safety nets
+func (r *Disruption) initialSafetynets() ([]string, error) {
+	responses := []string{}
+	// handle initial safety nets if safemode is enabled
+	if r.Spec.Unsafemode == nil || !r.Spec.Unsafemode.DisableAll {
+		if caught, err := safetyNetCountNotTooLarge(*r); err != nil {
+			return nil, fmt.Errorf("error checking for countNotTooLarge safetynet: %w", err)
+		} else if caught {
+			logger.Debugw("the specified count represents a large percentage of targets in either the namespace or the kubernetes cluster", r.Name, "SafetyNet Catch", "Generic")
+
+			responses = append(responses, "the specified count represents a large percentage of targets in either the namespace or the kubernetes cluster")
```

Comment:
Could `safetyNetCountNotTooLarge` return more info than just a boolean, so we could let the user know exactly how many targets were selected compared to the safe limit?